apps/api/src/app/controllers/__tests__/createTestModule.ts

```diff
@@ -73,9 +73,11 @@ import {
     CreateDigitalTextCommandHandler,
     DigitalTextCreated,
     DigitalTextPageContentTranslated,
+    DigitalTextTitleTranslated,
```

Comment:
I'm going to try to help you avoid using `createTestModule` for the test setup on this one.

Reply:
Hit some issues. We definitely need to fight this battle soon, though.